config/crd/bases/dscinitialization.opendatahub.io_dscinitializations.yaml

```diff
@@ -115,6 +115,38 @@ spec:
                     x-kubernetes-validations:
                     - message: MonitoringNamespace is immutable
                       rule: self == oldSelf
+                  traces:
```

Comment:
This is the thing I mentioned in chat.

These monitoring APIs being in the monitoringCommonSpec, mean we end up with this - the content being duplicated in the DSCI and the monitoring CR.

It's cleaner for there to only be one source of truth, when configuring your monitoring you should go to the monitoring CR.